types/proposal.ts

```diff
@@ -1,30 +1,36 @@
 import { z } from 'zod'
 
-const Proposal = z.object({
+const ProposalResponse = z.object({
 	id: z.number().int().positive(),
-	created: z.string(),
-	updated: z.string(),
+	created: z.number(),
```

Comment:
change/question: we should still transform these. However we want to schema/structure it in the backend is fine but in the response to the frontend we should send ISO 8601. JS sucks at everything else and it's the most reliable standard. PG will output ISO8601 as one of the standard outputs of a timestamptz data type so this shouldn't be something we have to work around. I'm wondering:
A) what the value is of the `datestyle` setting in the postgres container - ISO8601 is the default I believe, but it will output a few standard flavors directly. 
B) If we need to explicitly set it in the docker compose like [this](https://stackoverflow.com/a/78478688/13393468) 
C) I thought we had this working previously returning a string prior to these PR changes. IIRC the issue was with the default datetime parser for slonik being integer and needing to not use that one. This means we probably need to register the type parsers more selectively in the slonik config but not positive.